### ISSUE TITLE
FTP connection error

### ISSUE THREAD

**Issue by vol4ikman** [theproducer/brackets-ftp#5]
Hi, thank you for plugin, but I can't connect to my server. I enter my server address, username, pass and root dir -> click on "Save Settings" and nothing.....

I use Linux Mint 15 32bit

any ideas?

thank you

**Comment by theproducer** [comment]
Yes, its not supposed to display a file browser or anything.  After you've saved your connection settings, files that you edit and save in Brackets will automatically be uploaded into the root directory that you set (mirroring the directory structure of the Brackets project).

I know I need to write some better documentation to help explain.  I am working on adding a traditional FTP file browser which should be available in a few weeks.

**Comment by Wikunia** [comment]
+1 sounds great!
